src/harness/testcases/WINNF_FT_S_FDB_testcase.py

```diff
@@ -1465,3 +1469,10 @@ def test_WINNF_FT_S_FDB_8(self, config_filename):
     # Check the heartbeat response code is 500(TERMINATED_GRANT)
     for resp in heartbeat_responses:
       self.assertEqual(resp['response']['responseCode'], 500)
+
+  def check_default_test_not_expired(self):
+    """Check the expiration date in the default configs has not been reached."""
+    if datetime.now() > datetime.strptime('2028-09-23', '%Y-%m-%d') - timedelta(days=1):
+      logging.error('Test is being run near or past the expiration date of the '
```

Comment:
How about putting this as `logging.warn` instead of `error`?

Reply:
If this condition is ever true it's a very strong indication that you should likely not be running this test at all anymore hence the error level logging.